db/migrate/20180809101455_merge_duplicate_schools.rb

```diff
@@ -0,0 +1,29 @@
+class MergeDuplicateSchools < ActiveRecord::Migration
+  def up
+    return if School.count == 0 # for Travis
+
+    # This list of schools comes from the spreadsheet at https://jira.nypl.org/browse/MLN-162
+    # format of each school_id_array is [id_of_school_to_merge, id_of_school_to_keep]
```

Comment:
What does school_to_merge man?

Reply:
The `school_to_merge` is going to be deleted, so we will merge any data about that school into the `school_to_keep`.  This includes patrons assigned to that school and the `campus_id`.